moon-store-mac-os/moon-store-mac-os/Domain/Repositories/AuthenticationRepository.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  LoginRepository.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 12/14/24.
+//
+
+import Foundation
+
+final class AuthenticationRepository: BaseNetworkService {
+    var loggedUser: UserModel? {
+        guard let storedUser = try? store.fetch().first else { return nil }
+        return storedUser
+    }
+    
+    var isLoggedUser: Bool { loggedUser != nil }
+    
+    private let decoder: JSONDecoder = .init()
+    private let store: DataManager<UserModel> = .init()
```

Comment:
Use computed var after other simple declaration var

```suggestion  
    private let decoder: JSONDecoder = .init()
    private let store: DataManager<UserModel> = .init()
    
     var loggedUser: UserModel? {
        guard let storedUser = try? store.fetch().first else { return nil }
        return storedUser
    }
    
    var isLoggedUser: Bool { loggedUser != nil }
```